Tools/DeployApps/BaasClient.cs

```diff
@@ -863,6 +881,180 @@ private static class Schemas
             },
             GenericBaasRule(differentiator, "foos"));
         }
+
+        private class BaaSaasClient
```

Comment:
I would probably call it `Baasaas` - capitalizing the `S` makes it super weird, especially in camelcase, where it looks like the splitting is `baa saa s` 😅